README.md

```diff
@@ -56,24 +56,23 @@ Partially implement of specification: [https://github.com/open-telemetry/opentel
 ### Create a Context
 
 ```lua
-local context_storage = require("opentelemetry.context_storage")
--- context_storage is a centralized storage in request scope, so you can get current Context in different request phase.
-local context = require("opentelemetry.context").new(context_storage)
+-- context is stored on ngx.ctx by default, so context is passed between different request phases.
+local context = require("opentelemetry.context").new()
 ```
 
 ### Attach/Detach Context
 
 ```lua
--- Associates a Context with the caller's current execution unit, so you can use context:current() to retrieve it.
-context:attach()
--- Resets the Context associated with the caller's current execution unit to the value it had before attaching a specified Context.
-context:detach()
+-- Associates a Context with the caller's current execution unit, so you can use context:current() to retrieve it. Returns a token you can use to detach.
```

Comment:
Suggest replace all `context:current()` to `context.current()`, make call style consistently.

Reply:
👍 eb4c044134509deaab68efa9bf56f30214be4e2d